docs/install-and-setup.md

```diff
@@ -217,44 +216,56 @@ well as some command-specific ones like `--revision`, `--from` and `--to`. You
 can activate them with the alternative "dynamic" instructions below. They should
 still complete everything the static completions did, so only activate one of
 them. Please let us know if you encounter any issues, so we can ensure a smooth
-transition once we default to these new completions. Our initial experience
-is that these new completions work best with `fish`. If you have ideas about
-specific completions that could be added, please share them
-[here](https://github.com/jj-vcs/jj/issues/4763).
+transition once we default to these new completions.
+
+!!! Info
```

Comment:
You could use `> [!NOTE]` or similar here to make it stand out visually:
https://github.com/orgs/community/discussions/16925

> In recent times multiple users showed up and were confused
As one of these people this PR would not have helped me because I didn't manually install completions, they were provided by my distro package. 

But still these changes look like an improve in clarity of the docs 👍

Reply:
Possibly these should be admonitions in this style to make it render well on the website? https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
Yes, it is supposed to be a mkdocs adminition, to thank you for linking it.

Reply:
I've adjusted it to the correct format now.